lib/form/html.js

```diff
@@ -90,6 +90,7 @@ const addRequired = (html, input, index) => {
 
     if (input.required === 'save' || inp.required === 'save' || input.required === 'publish' || inp.required === 'publish') {
       let level = '';
+      const required = input.required && inp.required;
```

Comment:
`&&` is not what you want, it would be `||` actually, this should be:
`const required = inp.required || input.required`
then you can get rid of the whole conditional:
```
const level = `required--${required}`
```